src/App.css

```diff
@@ -2,11 +2,30 @@
   text-align: center;
 }
 
+.container {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
```

Comment:
Even after providing width and height as 100% why you are still using top, bottom, left, right any specific reason. 

If possible can you please share live preview link